src/components/MessageLogger/LineMessage.tsx

```diff
@@ -24,19 +52,40 @@ export const LineMessage = ({ message, count }: Props) => {
           whiteSpace: isOpen ? "normal" : "nowrap",
         }}
       >
-        <IoIosArrowDropdownCircle
+        <BsFillCaretRightFill
           id={styles.dropDownArrowIcon}
           onClick={dropDownLine}
           style={{
-            transform: isOpen ? "none" : "rotate(270deg)",
+            transform: isOpen ? "rotate(90deg)" : "none",
+            visibility: !isOverflowing ? "hidden" : "visible",
           }}
         />
-        <label id={styles.idMsg}>{message.id}: </label>
+        <label
+          id={styles.idMsg}
+          style={{
+            color: message.type === "warning" ? "#ce980e" : "#c51010",
```

Comment:
Instead of picking the color here, pass a color prop to the component. That way, you can use any color and change it easily from outside. Do it in way that you can generate the background color from the text color. I recommend that you look into HSL color, which lets you easily change the lightness of a color.